### PR TITLE
Engdocs 1941

### PR DESCRIPTION
**Pull request by aevesdocker**
Part of https://www.notion.so/dockerinc/Compose-spec-deprecation-427b924a1aa0435e959d68ddd8716dbb

This simple PR moves the content into our repo. More to follow. 

cc @glours @ndeloof

### CODE REVIEW

**Comment by dvdksn** [comment]
I don't think the data file for `docker init` is maintained in an upstream repo.

https://github.com/docker/docs/blob/766c54fe380317cefeb33b182112b3cc8a084721/data/init-cli/docker_init.yaml

cc @craig-osterhout

**Comment by craig-osterhout** [comment]
> I don't think the data file for `docker init` is maintained in an upstream repo.
> 
> https://github.com/docker/docs/blob/766c54fe380317cefeb33b182112b3cc8a084721/data/init-cli/docker_init.yaml
> 
> cc @craig-osterhout

Correct, it's currently not maintained upstream.

**Comment by dvdksn** [comment]
Could you rebase so that the vendor commit (fd6fb9c77b31013d9afa5e6ba41ec872bad8c40a) is in a separate commit, and the other commits are squashed?

**Comment by dvdksn** [comment]
A squashed diff will show these files as having been moved from the _vendor dir to the content dir. Also I see that the content is different in these files. Maybe we should instead do this in three commits

- update the compose import first so it's on the latest commit
- then remove the vendoring
- add the files, ideally preserving history?

**Review by ndeloof — APPROVED** [review]

**Review by usha-mandya — DISMISSED** [review]
Thanks for moving forward with this update @aevesdocker. Maintaining the upstream compose-spec will involve a wider discussion. Let's go ahead with the docs changes so we are not blocked on this.

File: content/compose/compose-file/05-services.md [hunk 1]
```diff
@@ -1,166 +1,8 @@
...
-  - NET_ADMIN
-  - SYS_ADMIN
-```
-
-## cgroup
+---
+title: Services top-level elements
+description: Explore all the attributes the services top-level element can have.
+keywords: compose, compose specification, services, compose file reference
+---
```
Comment (glours) [inline content/compose/compose-file/05-services.md:5]: Why do we lost all the attributes from `attach` to `cap_drop` during the move of the file? Also the title for the `cgroup` argument is missing

**Review by glours — CHANGES_REQUESTED** [review]
I think we have lost parts of the `services.md` file

**Review by glours — APPROVED** [review]
LGTM

**Review by usha-mandya — DISMISSED** [review]

**Comment by usha-mandya** [comment]
:shipit:

**Review by usha-mandya — APPROVED** [review]

**Pull request merged**